lib/checking_account.rb

```diff
@@ -0,0 +1,62 @@
+module Bank
+  class CheckingAccount < Account
+    attr_accessor :counter
+    def initialize(id, balance, date = nil )
+      super(id, balance, date)
+      @counter = 0 # Keeps track of number of check transactions
```

Comment:
`@counter` is a pretty generic variable name - could you have called this something more descriptive? Maybe `@checks_used`?